### PR TITLE
Chore/dep upgrades

### PR DESCRIPTION
**Pull request by danethurber**
update test, lint, and storybook dependencies.

there are some deprecation warnings we'll need to silence in the future

### CODE REVIEW

File: docs/gatsby-config.js [hunk 1]
```diff
@@ -5,6 +5,12 @@ const path = require('path')
 const remarkSlug = require('remark-slug')
 
 module.exports = {
+  flags: {
+    DEV_SSR: true,
+    FAST_DEV: true,
```
Comment (jaketrent) [inline docs/gatsby-config.js:10]: this is experimental, right?

File: packages/actionmenu/jest.config.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 const baseConfig = require('../../jest/base.config.js')
-const { name } = require('./package')
+const { name } = require('./package.json')
 
 module.exports = {
   ...baseConfig,
   displayName: name,
   name: name,
-  testMatch: [`${__dirname}/**/*/?(*.)+(spec|test).(ts|tsx)`]
+  testMatch: [path.join(__dirname, '/**/*/?(*.)+(spec|test).(js|ts|tsx)')]
```
Comment (jaketrent) [inline packages/actionmenu/jest.config.js:10]: really not a great reason to *partially* use path.join.  It'll be OS-specific only for that first separator.  But we still have forward-slashes hardcode after that.

**Review by jaketrent — APPROVED** [review]

File: packages/actionmenu/jest.config.js (hunk 2 above)
Comment (danethurber) [reply packages/actionmenu/jest.config.js:10]: yeah, it's some new linting rule. i think it's when used with `__dirname` 🤷

File: docs/gatsby-config.js (hunk 1 above)
Comment (danethurber) [reply docs/gatsby-config.js:10]: yes. gatsby gave me a message that i should try these out...so i'm trying them out :)

**Pull request merged**